services/auth.js

```diff
@@ -90,7 +91,41 @@ const authService = {
     await tokenService.saveToken(user._id, tokens.refreshToken)
 
     return tokens
-  }
+  },
+
+  sendResetPasswordEmail: async (email, refreshToken) => {
```

Comment:
why pass there refreshToken?
refreshToken is used to retrieve access token

Reply:
Now edited, used reset tokens